Src/AutoFakeItEasy/Properties/AssemblyInfo.cs

```diff
@@ -33,8 +33,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("3.50.2.0")]
-[assembly: AssemblyFileVersion("3.50.2.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyInformationalVersion("1.0.0.0")]
```

Comment:
>I'd say that confusion should not be an issue. We just need to put a notice somewhere in the wiki

Or maybe on the README.md. My only concern with the wiki is that it's not part of the Git repository, it's a _GitHub_ thing (I could be wrong though).